test/nodes/anomaly_test.rb

```diff
@@ -33,6 +34,12 @@ def test_should_filter_by_no_postnatal_anomaly
     assert_equal({ Canql::EQUALS => false }, parser.meta_data['anomaly.postnatal.exists'])
   end
 
+  def test_should_filter_by_no_postnatal_anomaly_and_prenatal_anomalies
+    parser = Canql::Parser.new('all cases with no postnatal anomalies and prenatal anomalies')
+    assert parser.valid?
+    assert_equal({ Canql::EQUALS => false }, parser.meta_data['anomaly.postnatal.exists'])
```

Comment:
Should we have an additional assertion here that is checking:

```ruby
assert_equal({ Canql::EQUALS => false }, parser.meta_data['anomaly.prenatal.exists'])
```

... as I think the language is ambiguous whether the intent is:

```
A: all cases with (no  postnatal anomalies) and (prenatal anomalies)
B: all cases with  no (postnatal anomalies  and  prenatal anomalies)
```